pkgs/by-name/jj/jjui/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  pkgs,
+  lib,
+  ...
+}:
+pkgs.buildGoModule rec {
+  pname = "jjui";
+  version = "b1f467fcef89a6f9e57a5be2108bb0ef34f1442d";
```

Comment:
Per our [versioning guidelines](https://github.com/NixOS/nixpkgs/tree/128bd7f9742a0adfef70142d5b093e9f8383aef3/pkgs#versioning):

> - If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format.
>
> Example: Given a project had its latest releases 2.2 in November 2021 and 3.0 in January 2022, a commit authored on March 15, 2022 for an upcoming bugfix release 2.2.1 would have version = "2.2-unstable-2022-03-15".
>
> - If a project has no suitable preceding releases - e.g., no versions at all, or an incompatible versioning or tagging scheme - then the latest upstream version in the above schema should be 0.
>
> Example: Given a project that has no tags or released versions at all, or applies versionless tags like latest or YYYY-MM-DD-Build, a commit authored on March 15, 2022 would have version = "0-unstable-2022-03-15".

In this case it should be `0-unstable-2024-11-28`

Reply:
Aha. TIL. Thank you for the thorough review. Fixed.